src/librustc_mir/build/mod.rs

```diff
@@ -89,6 +89,25 @@ pub fn mir_build(tcx: TyCtxt<'_>, def_id: DefId) -> Body<'_> {
                 _ => None,
             };
 
+            // if this fn has #[track_caller], it will receive an implicit argument with a location
```

Comment:
Ideally, this construct should be extracted out to a function `-> Option<ArgInfo>`.

Reply:
Sure, keeping it adjacent to the current method seems good readability-wise.